src/lib/swapper/swappers/CowSwapper/CowSwapper.test.ts

```diff
@@ -66,7 +66,7 @@ describe('CowSwapper', () => {
 
   describe('getType', () => {
     it('returns the correct type for CowSwapper', () => {
-      expect(swapper.getType()).toEqual(SwapperType.CowSwap)
+      expect(swapper.getType()).toEqual(SwapperType.CowSwapEthereum)
     })
   })
 
```

Comment:
<i class='graphite__hidden'>[Re: lines 99 to 99]<br/></i>

Can also test XDAI<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/shapeshift/web/4595?utm_source=unchanged-line-comment">Graphite</a>.</span>